.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
Commit message: Please move your sign-off below the commit body. And move the `Fix #6540` to the commit body as `Fixes #6540`, then GitHub will automatically close this issue.